### ISSUE TITLE
pretrained

### ISSUE THREAD

**Issue by 06Liz** [BioMedAI-UCSC/InverseSR#10]
const.py文件中，import os
from pathlib import Path

# Use environment variables to auto-detect whether we are running an a Compute Canada cluster:
# Thanks to https://github.com/DM-Berger/unet-learn/blob/master/src/train/load.py for this trick.
COMPUTECANADA = False
TMP = os.environ.get("SLURM_TMPDIR")

if TMP:
    COMPUTECANADA = True

if COMPUTECANADA:
    INPUT_FOLDER = Path(str(TMP)).resolve() / "work" / "inputs"
    MASK_FOLDER = Path(str(TMP)).resolve() / "work" / "inputs" / "masks"
    PRETRAINED_MODEL_FOLDER = Path(str(TMP)).resolve() / "work" / "trained_models"
    PRETRAINED_MODEL_DDPM_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "ddpm"
    )
    PRETRAINED_MODEL_VAE_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "vae"
    )
    PRETRAINED_MODEL_DECODER_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "decoder"
    )
    PRETRAINED_MODEL_VGG_PATH = (
        Path(str(TMP)).resolve() / "work" / "trained_models" / "vgg16.pt"
    )
    OUTPUT_FOLDER = Path(str(TMP)).resolve() / "work" / "outputs"
else:
    INPUT_FOLDER = Path(__file__).resolve().parent.parent.parent / "data" / "IXI"
    MASK_FOLDER = Path(__file__).resolve().parent.parent / "masks"
    OASIS_FOLDER = Path(__file__).resolve().parent.parent.parent / "data" / "OASIS"
    PRETRAINED_MODEL_FOLDER = (
        Path(__file__).resolve().parent.parent.parent / "data" / "trained_models"
    )这些预训练的模型都在哪里啊，数据在哪啊？

**Comment by 06Liz** [comment]
而且那个链接也打不开

**Comment by jueqiw** [comment]
Thanks for your interest in our work! The weight for the pre-trained model could be found in https://drive.google.com/drive/folders/110l68um6gUJzECIv0AyF-4Fcw0rrQgA9?usp=drive_link

Please don't hesitate to let me know if you have future questions!

**Comment by 06Liz** [comment]
Thanks for your answer and your contribution.

**Comment by 06Liz** [comment]
with open(
        "/scratch/j/jlevman/jueqi/thesis_experiments/decoder/result_decoder_downsample_2.csv",
        "a",
    ) as file:
        writer = csv.writer(file)
        writer.writerow(row)

    return latent_vector_out
 @wendy-xiaozong 
    Where is the /scratch/j/jlevman/jueqi/thesis_experiments/decoder/result_decoder_downsample_2.csv?

**Comment by jueqiw** [comment]
It's an output file that I used to save the result
